### ISSUE TITLE
Question about darkness_adaptivity

### ISSUE THREAD

**Issue by 519174419** [colmap/colmap#1943]
Firstly, Thanks for the great project for SfM!!!

And I found a flag named "darkness_adaptivity" in sift.h, as :
  // Whether to adapt the feature detection depending on the image darkness.
  // Note that this feature is only available in the OpenGL SiftGPU version.
  bool darkness_adaptivity = false;

Is it flase defaultly, My question is weather its useful for the robustness of Sfm in dark envirement~

**Comment by ahojnnes** [comment]
Yes, it can help in some cases, but I have not played with this parameter for a long time.

**Comment by ahojnnes** [comment]
Closing: darkness_adaptivity flag explained by maintainer.
